### ISSUE TITLE
Dataset Issue

### ISSUE THREAD

**Issue by ycjing** [harveyslash/Deep-Steganography#4]
Nice implementation! I wonder how you selected your training and testing images. Thanks!

**Comment by harveyslash** [comment]
Training and testing images were selected at random from imagenet

**Comment by anilsathyan7** [comment]
Hi, how many images have you used for training ? Also it looks like you have run the training for 24 thousand epoch?? 
I tried the same network  (keras) with 50k images.However, i'am confused how long (and how much images) i should train, since it takes 90 minutes for 1 epoch...
Also even though validation loss goes to 0.0069,a green outline of secret image seems to be always visible in the container image(i tried upto 20 epochs with this setup) ...
Here is my network...Looks like i'am missing something !!!
![mystegnet](https://user-images.githubusercontent.com/1130185/61739098-62ca1580-ada9-11e9-80eb-076375aaa613.png)
